admin/common_auth/logs.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.admin.models import LogEntry, CHANGE, LogEntryManager
+
+ACCEPT_PREREG = 10
+REJECT_PREREG = 11
+COMMENT_PREREG = 12
+
+CONFIRM_SPAM = 20
+CONFIRM_HAM = 21
+
+
+def update_admin_log(user_id, object_id, object_repr, message, action_flag=CHANGE):
+    try:
+        OSFLogEntry.objects.log_action(
+            user_id=user_id,
+            content_type_id=None,
+            object_id=object_id,
+            object_repr=object_repr,
+            change_message=message,
+            action_flag=action_flag
+        )
+    except:
+        print 'Failed to log changes to {}'.format(object_id)
+
+
+class OSFLogEntryManager(LogEntryManager):
```

Comment:
The `OSFLogEntry` class did not like it when it didn't have its very own Manager class rather than just using the inherited class. Hence why this exists.